app/services/mqtt_controller_service.rb

```diff
@@ -9,7 +9,9 @@ def initialize(options)
   end
 
   def process
-    system(publish)
+    Rails.logger.info("publish: #{publish}")
```

Comment:
@hsong-rh We need a # TODO to indicate this is a temporary solution and wont work in the CI env. There we would have to go thru the Cloud Controller.